docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md

```diff
@@ -5,17 +5,17 @@ pagename: setting-python-3-desktop
 lang: en
 ---
 
-# Set Python 2 as the default Python version in {% include product %} Desktop
+# Set Python 3 as the default Python version in {% include product %} Desktop
 
-**Note:** Python 2 is getting removed by November 1st 2022. You can find the notification [here](https://community.shotgridsoftware.com/t/important-notice-upcoming-removal-of-python-2-7-and-3-7-interpreter-in-shotgrid-desktop/15166).
+**Important:** **This topic is only valid when using {% include product %} Desktop version 1.7.3**—these steps are no longer needed when using newer versions of {% include product %} Desktop. Python 2 was removed November 1st 2022 due to security reasons, [learn more here](https://community.shotgridsoftware.com/t/important-notice-upcoming-removal-of-python-2-7-and-3-7-interpreter-in-shotgrid-desktop/15166).
```

Comment:
The date is incorrect. We removed Python 2 from SGD on Jan. 26, 2023 with the release of SGD 1.8.0.